R/api.R

```diff
@@ -72,24 +75,27 @@ metadata <- function() {
 
 #' @porcelain
 #'   POST /scenario/run => json(scenarioRunResponse)
+#'   state queue :: queue
 #'   body data :: json(scenarioRunRequest)
-scenario_run <- function(data) {
-  # Returning sample response for now
-  json_verbatim(read_local_json("sample_scenario_run_response.json"))
+scenario_run <- function(queue, data) {
+  data <- jsonlite::parse_json(data)
+  run_id <- queue$queue_model_run(
+    data$parameters,
+    model_version = data$modelVersion
+  )
+  json_verbatim(list(runId = run_id))
```

Comment:
I don't think you want json_verbatim here but a conversion to json? this would hopefully become apparent with a test :)

Reply:
This is getting tested by the e2e test but i guess not detected by codecov..

The `json_verbatim` util does the auto unboxing, as I was finding that without that, the runId value got wrapped in an array. Should I be doing something else to stop that from happening? What do you mean by conversion to json? Using jsonlite or something else?

Reply:
```suggestion
  jsonlite::toJSON(list(runId = jsonlite::unbox(run_id)))
```

or more simply:

```suggestion
  list(runId = jsonlite::unbox(run_id))
```